MinVer/MinVer.csproj

```diff
@@ -9,26 +9,21 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <!-- NU5100: The folder being packaged contains an assembly file that is not in lib folder. -->
     <NoWarn>$(NoWarn);NU5100</NoWarn>
-    <OutputType>Exe</OutputType>
     <PackageIcon>minver.png</PackageIcon>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/adamralph/minver</PackageProjectUrl>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageReleaseNotes>https://github.com/adamralph/minver/blob/main/CHANGELOG.md</PackageReleaseNotes>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <RollForward>major</RollForward>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
-    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
     <LangVersion>default</LangVersion>
+    <Version>6.0.0</Version>
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\minver-cli\Options.cs" Link="Options.cs" />
-    <Compile Include="..\minver-cli\Program.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="4.1.0" PrivateAssets="All" Publish="true" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />

```

Comment:
@bording are you sure we need this? I tried building without it and the `AssemblyInformationalVersion` had the commit SHA added, so I think just using the .NET 8 SDK to build the project is enough, even though the project targets `net6.0`.

Reply:
At the time of the spike, you had not moved the repo over to the .NET 8 SDK. You're correct that you don't need it anymore.

Reply:
Ah OK, I misunderstood the effect of the cherry-pick. I thought the commit in your spike branch was adding this line.